system/modules/core/elements/ContentAlias.php

```diff
@@ -39,7 +39,7 @@ public function generate()
 			return '';
 		}
 
-		// Overwrite the ID
+		/** @var \ContentElement $objElement */
```

Comment:
Yes, but `$objElement` is a content model first and then a content element. Ugly, I know :(